topics-project/__e2e__/profile.spec.js

```diff
@@ -0,0 +1,77 @@
+// @ts-check
+
+import { test, expect } from "@playwright/test";
+
+test("page successfully fetches user", async ({ page }) => {
+  await page.goto("http://localhost:3000/login");
+  await page.getByPlaceholder("Enter email").click();
```

Comment:
Not required but a suggestion would be if you are gonna call the same thing 3 times (`page.getByPlaceholder("Enter email")`) then prob be best to make it a variable and just call that variable each time instead.

Reply:
Okay